### PR TITLE
Correct some ES terms

### PR DESCRIPTION
**Pull request by stevemao**
The latest is ECMAScript 2016

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 
-ECMAScript2015(ES6) is the latest version of the ECMAScript standard. 
+ECMAScript 2016 (ES7) is the latest version of the ECMAScript standard.
```
Comment (cht8687) [inline README.md]: I think we are emphasising on ES6 features here. Could you think about a better way for this?

Comment (cht8687) [inline README.md]: Check @gaearon 's comments in [here](https://github.com/cht8687/React-ES5-To-ES6-Checklist/issues/4), 

> ES7 are experimental language proposals that may or may not get into the language someday.'

Comment (stevemao) [inline README.md]: No, ES7 is the latest version (only two features added). We could say something like "ES6 and beyond is popular"

Comment (cht8687) [inline README.md]: This sounds a good word to me.

**Comment by stevemao** [comment]
I'm casting my magic Dan Abramov card

image

**Pull request merged**